<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedGridNewDialog</class>
 <widget class="QDialog" name="LedGridNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>168</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>168</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>168</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建LED</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
  font-family: &quot;Arial&quot;, &quot;Segoe UI&quot;, &quot;黑体&quot;, &quot;等线&quot;;
  color: white;
  background: rgb(50, 50, 50);
}
*:!enabled {
  color: rgb(155, 155, 155);
}

#codeBrowser {
  font-family: &quot;Consolas&quot;;
}
#mainColor {
  background-color: #fff;
}
#subColor {
  background-color: #000;
}
/**********子界面背景********* */

QDockWidget &gt; QWidget {
  background: rgb(75, 75, 75);
}
QDockWidget,
QLabel {
  font-size: 14px;
  color: white;
}
QDockWidget::title {
  text-align: center;
  background: rgb(90, 90, 90);
}

QLabel {
  background: transparent;
}
/**********提示********* */
QToolTip {
  color: yellow;
  border: 2px solid #333;
}

/**********菜单********* */
QMenu {
  border: 1px solid rgb(100, 100, 100);
  background: rgb(68, 69, 73);
}
QMenu::item {
  height: 22px;
  padding: 0px 25px 0px 20px;
}
QMenu::item:enabled {
  color: rgb(225, 225, 225);
}
QMenu::item:enabled:selected {
  color: rgb(230, 230, 230);
  background: rgba(255, 255, 255, 80);
}
QMenu::separator {
  height: 1px;
  background: rgb(100, 100, 100);
}
QMenu::indicator {
  width: 13px;
  height: 13px;
}
QMenu::icon {
  padding-left: 2px;
  padding-right: 2px;
}
/**********分组框********* */
QGroupBox {
  font-size: 12px;
  border: 1px solid rgb(80, 80, 80);
  border-radius: 4px;
  margin-top: 10px;
}
QGroupBox::title {
  color: rgb(175, 175, 175);
  top: -12px;
  left: 10px;
}

/**********页签项********* */
QTabWidget::pane {
  border: none;
  border-top: 3px solid rgb(0, 160, 230);
  background: rgb(57, 58, 60);
}
QTabWidget::tab-bar {
  border: none;
}
QTabBar::tab {
  border: none;
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
  color: rgb(175, 175, 175);
  background: rgb(255, 255, 255, 30);
  height: 28px;
  min-width: 85px;
  margin-right: 5px;
  padding-left: 5px;
  padding-right: 5px;
}
QTabBar::tab:hover {
  background: rgb(255, 255, 255, 40);
}
QTabBar::tab:selected {
  color: white;
  background: rgb(0, 160, 230);
}

/**********表头********* */
QHeaderView {
  border: none;
  border-bottom: 3px solid rgb(0, 160, 230);
  background: rgb(57, 58, 60);
  min-height: 30px;
}
QHeaderView::section:horizontal {
  border: none;
  color: white;
  background: transparent;
  padding-left: 5px;
}
QHeaderView::section:horizontal:hover {
  background: rgb(0, 160, 230);
}
QHeaderView::section:horizontal:pressed {
  background: rgb(0, 180, 255);
}

/**********表格********* */
QTableView,
QTreeView,
QListView {
  border: 1px solid rgb(45, 45, 45);
  background: rgb(57, 58, 60);
  gridline-color: rgb(60, 60, 60);
}
QTableView::item,
QTreeView::item,
QListView::item {
  padding-left: 5px;
  padding-right: 5px;
  border: none;
  background: rgb(72, 72, 74);
  border-right: 1px solid rgb(45, 45, 45);
  border-bottom: 1px solid rgb(45, 45, 45);
}
QTableView::item:selected,
QTreeView::item:selected,
QListView::item:selected {
  background: rgba(202, 81, 0, 50);
}
QTableView::item:selected:!active,
QTreeView::item:selected:!active,
QListView::item:selected:!active {
  color: white;
}
QTableView::indicator,
QTreeView::indicator,
QListView::indicator {
  width: 20px;
  height: 20px;
}

/**********滚动条-水平********* */
QScrollBar:horizontal {
  height: 20px;
  background: transparent;
  margin-top: 3px;
  margin-bottom: 3px;
}
QScrollBar::handle:horizontal {
  height: 20px;
  min-width: 30px;
  background: rgb(68, 69, 73);
  margin-left: 15px;
  margin-right: 15px;
}
QScrollBar::handle:horizontal:hover {
  background: rgb(80, 80, 80);
}
QScrollBar::sub-line:horizontal {
  width: 15px;
  background: transparent;
  subcontrol-position: left;
}
QScrollBar::add-line:horizontal {
  width: 15px;
  background: transparent;
  subcontrol-position: right;
}
QScrollBar::sub-line:horizontal:hover {
  background: rgb(68, 69, 73);
}
QScrollBar::add-line:horizontal:hover {
  background: rgb(68, 69, 73);
}
QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
  background: transparent;
}

/**********滚动条-垂直********* */
QScrollBar:vertical {
  width: 20px;
  background: transparent;
  margin-left: 3px;
  margin-right: 3px;
}
QScrollBar::handle:vertical {
  width: 20px;
  min-height: 30px;
  background: rgb(68, 69, 73);
  margin-top: 15px;
  margin-bottom: 15px;
}
QScrollBar::handle:vertical:hover {
  background: rgb(80, 80, 80);
}
QScrollBar::sub-line:vertical,
QScrollBar::add-line:vertical {
  border: 2px solid rgb(90, 90, 90);
}
QScrollBar::sub-line:vertical:hover,
QScrollBar::add-line:vertical:hover {
  background: rgb(68, 69, 73);
}
QScrollBar::up-arrow:vertical,
QScrollBar::down-arrow:vertical {
  width: 14px;
  height: 20px;
  background: rgb(90, 90, 90);
}
QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
  background: none;
}

/**********下拉列表********* */
QComboBox {
  height: 25px;
  border-radius: 4px;
  border: 1px solid rgb(100, 100, 100);
  background: rgb(72, 72, 73);
}
QComboBox:enabled {
  color: rgb(175, 175, 175);
}
QComboBox:enabled:hover,
QComboBox:enabled:focus {
  color: rgb(230, 230, 230);
  background: rgb(68, 69, 73);
}
QComboBox::drop-down {
  width: 20px;
  border: none;
  background: transparent;
}
QComboBox::drop-down:hover {
  background: rgba(255, 255, 255, 30);
}
QComboBox QAbstractItemView {
  border: 1px solid rgb(100, 100, 100);
  background: rgb(68, 69, 73);
  outline: none;
}
QComboBox QAbstractItemView::item {
  height: 25px;
  color: rgb(175, 175, 175);
}
QComboBox QAbstractItemView::item:selected {
  background: rgba(255, 255, 255, 40);
  color: rgb(230, 230, 230);
}

/**********进度条********* */
QProgressBar {
  border: 3px solid rgb(68, 69, 73);
  text-align: center;
  color: white;
  background: rgb(50, 50, 50);
}
QProgressBar::chunk {
  background: rgb(0, 160, 230);
}

/**********复选框********* */
/**********单选框********* */
QCheckBox,
QRadioButton {
  spacing: 5px;
}
QCheckBox,
QCheckBox {
  color: #afafaf;
}
QCheckBox:enabled:hover,
QRadioButton:enabled:hover {
  color: #c8c8c8;
  background-color: #666;
}
QCheckBox:enabled:checked,
QRadioButton:enabled:checked {
  color: green;
}
QCheckBox:enabled:checked:hover,
QRadioButton:enabled:checked:hover {
  color: green;
}
QCheckBox::indicator,
QRadioButton::indicator {
  width: 20px;
  height: 20px;
}

/**********输入框********* */
QLineEdit {
  height: 25px;
  border: 2px solid rgb(120, 130, 120);
  background: rgb(72, 72, 73);
}
QLineEdit:enabled {
  color: rgb(175, 175, 175);
}
QLineEdit:enabled:hover,
QLineEdit:enabled:focus {
  color: rgb(230, 230, 230);
  border: 2px solid rgb(140, 160, 180);
}

/**********文本编辑框********* */
QTextEdit {
  border: 1px solid rgb(45, 45, 45);
  color: rgb(175, 175, 175);
  background: rgb(57, 58, 60);
}

/**********滚动区域********* */
QScrollArea {
  border: 1px solid rgb(45, 45, 45);
  background: rgb(57, 58, 60);
}
QScrollBar::add-line:vertical {
  height: 0px;
  background-color: transparent;
  subcontrol-position: bottom;
}
QScrollBar::sub-line:vertical {
  height: 0px;
  background-color: transparent;
  subcontrol-position: top;
}

/**********按钮********* */
QPushButton {
  border: 2px solid rgb(85, 85, 85);
  width: 75px;
  height: 25px;
}
QPushButton:enabled {
  background: rgb(59, 59, 59);
  color: white;
}
QPushButton:checked {
  border-color: green;
}
QPushButton:enabled:hover {
  background: rgb(120, 120, 120);
}
QPushButton:enabled:pressed {
  background: rgb(90, 90, 90);
}
/******* ToolBox ******/
QToolBox::tab {
  border-width: 2px 0;
  border-style: solid;
  border-color: rgb(140, 160, 180);
}
QToolBox::tab:first {
  border-top-width: 0px;
}
QToolBox::tab:last {
  border-bottom-width: 0px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="pixelSizeLabel">
     <property name="text">
      <string>阵列大小</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="pixelHCount">
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="pixelVCount">
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="pixelSizeLabel_2">
     <property name="text">
      <string>常用</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="pixelSizeChoice">
     <item>
      <property name="text">
       <string>4x8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8x8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8x16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16x16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16x32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32x32</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LedGridNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
